pages/phases/getting-started/time-commitment.md

```diff
@@ -30,11 +30,11 @@ intro: The SME-QA process requires time commitments from subject matter experts,
           <dd class="chp-time-commitment__phase">Resume review training (3 hours) and interview training (2 hours)</dd>
           <dt class="chp-time-commitment__hours">5 hours (over 1 week)</dt>
           <dd class="chp-time-commitment__phase">Resume review</dd>
-          <dt class="chp-time-commitment__hours">20 hours split over three weeks</dt>
+          <dt class="chp-time-commitment__hours">14 hours split over three weeks</dt>
```

Comment:
why are we saying 3 weeks for resume review? am i misreading this? maybe we should just say- am for x number of weeks and the time per SME depends on how many SMEs and applicants you have.

Reply:
This text is for the assessments, not the resume review. Resume review is listed at five hours above: (150 applicants = 330 reviews, or 41.25 per SME. 7 min/review = 289 minutes = 4.81 hours.)

It always said three weeks. I just changed to "two weeks"

I modified to say that the hours are variable depending on the # passing resume review and round one: https://github.com/usds/SME-QA-Website/pull/243/commits/f672e37eab9a06920549b616c2606660657a0d47